apps/examples/rdp/rdp_main.c

```diff
@@ -99,13 +111,22 @@ static int rdp_demo(int argc, FAR char *argv[])
 		if (rdp_result != no_rdp_result) {
 			printf("rdp call fail!\n");
 			printf("rdp_result = 0x%x, no_rdp_result=0x%x\n", rdp_result, no_rdp_result);
-			goto end;
+			goto exit;
 		}
 	}
 	printf("rdp demo call succeed!\n");
-end:
+exit:
 	/* Frees the given secure context */
-	up_free_secure_context(tz_memory);
+	res = security_free(hnd, tz_memory);
```

Comment:
is tz_memory global variable? is it fine that application code can access this ?

Reply:
yes, tz_memory is global variable. Do you expect some problem to access it here?